molecule/default/molecule.yml

```diff
@@ -0,0 +1,19 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-ubuntu2004}-ansible:latest"
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
```

Comment:
Should we add an example of port forwarding here? [Example](https://github.com/Appsilon/ansible-rstudio-workbench/blob/main/molecule/default/molecule.yml)

Reply:
this is very specific to ansible roles that deploy server-based applications, additionally we don't know the port beforehand

Reply:
Yup, either we can add that to readme or let someone find it in the docs when he needs to.